res/info.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!-- Source: https://materialdesignicons.com/icon/information-outline -->
```

Comment:
That's just the SVG file header and a comment telling where the image was downloaded from.